bower.json

```diff
@@ -18,8 +18,10 @@
     "purescript-console": "^2.0.0",
```

Comment:
Can we also move purescript-console to dev dependencies? I worry why a CSV parser should need to write to the console :)